packages/babel-runtime-corejs2/regenerator/index.js

```diff
@@ -1 +1,15 @@
-module.exports = require("../helpers/regeneratorRuntime")();
+// TODO(Babel 8): Remove this file.
+
+const runtime = require("../helpers/regeneratorRuntime")();
```

Comment:
It is more appropriate to use `var` like runtime helpers, otherwise, this code needs to be compiled into es5 with babel self 👀

Reply:
Thanks 🚀